content/challenges/easy/4 - Gamble/func.js

```diff
@@ -12,5 +12,9 @@
  * @returns {boolean}
  */
 module.exports = (probabilityOfWinning, prize, costOfPlaying) => {
-    // your code here
+    if (probabilityOfWinning * prize - costOfPlaying > 0) {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
Whenever you have a function that is just returning `true` and `false` there's usually a meaningful simplification that can be done.

In this case I think you can just return the expression on line 15.
```suggestion
    return probabilityOfWinning * prize - costOfPlaying > 0;
```

since that equation itself evaluates to `true` or `false` as part of the logic you can just return the expression itself.